themes/mlibrary_new/exhibit-builder/exhibits/card.php

```diff
@@ -0,0 +1,16 @@
+<div class ="exhibit record">
+<?php
+     $title =  metadata($exhibit, 'title', array('snippet'=>300,'no_escape' => true));
```

Comment:
Do this lookup on the caller's side so that the template doesn't have to know how to lookup a title from an exhibit.